spec/factories/valid_draft_factories.rb

```diff
@@ -815,7 +819,7 @@
           'Eon' => 'test 3 Eon text 1'
         }]
       }],
-      'ISOTopicCategories' => %w(farming climatologyMeteorologyAtmosphere health)
+      'ISOTopicCategories' => %w[farming climatologyMeteorologyAtmosphere health]
```

Comment:
Thats not good. What version of Rubocop do you have? I have always seen it prefer `()`

I have 0.47.1